api/v1alpha5/metal3machine_types.go

```diff
@@ -62,6 +62,12 @@ type Metal3MachineSpec struct {
 	// NetworkData is an object storing the reference to the secret containing the
 	// network data given by the user.
 	NetworkData *corev1.SecretReference `json:"networkData,omitempty"`
+
+	// When set to disabled, automated cleaning will be skipped
```

Comment:
put the two lines together. The line break is affecting the description in the generated yaml file

Reply:
We already have such a descriptions that take more a than a single line, and I think this is okay because it is better to keep it short within the .go file which we read instead of auto-generated yaml file which we don't normally read